client/src/test/java/io/pravega/keycloak/client/KeycloakAuthzClientTest.java

```diff
@@ -67,16 +66,23 @@ public void getRPT_caching() {
         verify(tokenCache, times(1)).update(any());
     }
 
-    @Test(expected = KeycloakAuthenticationException.class)
+    @Test
     public void getRPT_error_authn() {
         AuthzClient client = mock(AuthzClient.class, Mockito.RETURNS_DEEP_STUBS);
         TokenCache tokenCache = spy(new TokenCache(0));
         when(client.obtainAccessToken()).thenThrow(new HttpResponseException("", 400, "", null));
 
         KeycloakAuthzClient authzClient = new KeycloakAuthzClient(client, tokenCache);
-        authzClient.getRPT();
+        try {
+            authzClient.getRPT();
+            Assert.fail();
+        }
+        catch(KeycloakAuthenticationException e) {
```

Comment:
Newline before catch is not needed, here and in other similar tests below.

Reply:
Fixed (i think I fixed all the places in the changed files that weren't confirming to it.